Chore: Added article links in italian footer

alevanni opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->

alevanni committed:
Update trending.json

Added 30 articles from the column "Published" on Trello

alevanni committed:
Merge branch 'freeCodeCamp:main' into it-footer

alevanni committed:
Merge branch 'main' into it-footer

alevanni committed:
Update trending.json

Replaced the first thirty titles and links

ieahleen commented on client/i18n/locales/italian/trending.json:
```diff
@@ -1,62 +1,61 @@
+"article5link":"https://www.freecodecamp.org/italian/news/linguaggi-di-programmazione-interpretati-e-compilati-qual-e-la-differenza/",
+"article6title":"L'Hook useEffect di React",
+"article6link":"https://www.freecodecamp.org/italian/news/hook-useeffect-react/",
+"article7title":"Donare un milione di dollari a freeCodeCamp",
```
questo forse è troppo lungo, il massimo sono 28 caratteri

ieahleen commented on client/i18n/locales/italian/trending.json:
```diff
@@ -1,62 +1,61 @@
+"article28title":"Elenco Keycode in JavaScript",
+"article28link":"https://www.freecodecamp.org/italian/news/elenco-codici-chiave-javascript/",
+"article29title":"Callback in JavaScript",
+"article29link":"https://www.freecodecamp.org/italian/news/funzioni-di-callback-javascript-cosa-sono-le-callback-in-js-e-come-usarle/"}
```
```suggestion
"article29link":"https://www.freecodecamp.org/italian/news/funzioni-di-callback-javascript-cosa-sono-le-callback-in-js-e-come-usarle/"
}

```

alevanni committed:
Merge branch 'freeCodeCamp:main' into it-footer

alevanni committed:
Update trending.json

Ho ridotto la lunghezza di alcuni titoli (<=28 caratteri) e spostato la parentesi di chiusura su una nuova riga

ieahleen commented on client/i18n/locales/italian/trending.json:
```diff
@@ -1,62 +1,62 @@
+"article3title":"Recuperare i dati in React",
+"article3link":"https://www.freecodecamp.org/italian/news/come-recuperare-i-dati-in-react-cheat-sheet-esempi/",
+"article4title":"Figure in CSS spiegate",
+"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/"
```
```suggestion
"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/",
```

ieahleen replied:
```suggestion
"article4link":"https://www.freecodecamp.org/italian/news/figure-in-css-spiegate/",
```

majestic-owl448 committed:
Add missing comma

majestic-owl448 committed:
JSON formatting

majestic-owl448 committed:
fixed titles length

nhcarrigan commented on client/i18n/locales/italian/trending.json:
```diff
@@ -1,62 +1,62 @@
+  "article26title": "Come commentare i JSON",
+  "article26link": "https://www.freecodecamp.org/italian/news/esempio-di-commento-json-come-commentare-nei-file-json/",
+  "article27title": "JavaScript Date Now",
+  "article27link": "freecodecamp.org/italian/news/javascript-date-now-come-ottenere-la-data-corrente-in-javascript/",
```
```suggestion
  "article27link": "https://www.freecodecamp.org/italian/news/javascript-date-now-come-ottenere-la-data-corrente-in-javascript/",
```

alevanni committed:
Apply suggestions from code review

Co-authored-by: Naomi Carrigan <nhcarrigan@gmail.com>
